for/for/ViewController.swift

```diff
@@ -27,12 +27,12 @@ class ViewController: UIViewController {
         super.viewDidLoad()
         
         for i in 1...10{
```

Comment:
とても細かいところで申し訳ないんだけど、Swiftでは中括弧の前はスペースを一つ開けるという慣習があります><


```suggestion
        for i in 1...10 {
```